src/main/java/de/thomaskrille/dropwizard_template_config/TemplateConfigBundleConfiguration.java

```diff
@@ -14,6 +16,7 @@
     private Optional<String> resourceIncludePath = Optional.absent();
     private Optional<String> fileIncludePath = Optional.absent();
     private Optional<String> outputPath = Optional.absent();
+    private Optional<Set<TemplateConfigProvider>> customProviders = Optional.absent();
```

Comment:
could we make it simpler by always having this guy initialized to an empty set and then not worry about Optional ?